[2024-06-25T21:26:08.326+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T21:26:08.599+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sp_budget_pipeline.ingestion-gdvDespesasExcel manual__2024-06-25T21:26:03.924987+00:00 [queued]>
[2024-06-25T21:26:08.623+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sp_budget_pipeline.ingestion-gdvDespesasExcel manual__2024-06-25T21:26:03.924987+00:00 [queued]>
[2024-06-25T21:26:08.643+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-25T21:26:09.324+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): ingestion-gdvDespesasExcel> on 2024-06-25 21:26:03.924987+00:00
[2024-06-25T21:26:09.344+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1870) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T21:26:09.348+0000] {standard_task_runner.py:63} INFO - Started process 1872 to run task
[2024-06-25T21:26:09.350+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sp_budget_pipeline', 'ingestion-gdvDespesasExcel', 'manual__2024-06-25T21:26:03.924987+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/sp_budget_pipeline.py', '--cfg-path', '/tmp/tmphal75kmb']
[2024-06-25T21:26:09.352+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask ingestion-gdvDespesasExcel
[2024-06-25T21:26:09.576+0000] {task_command.py:426} INFO - Running <TaskInstance: sp_budget_pipeline.ingestion-gdvDespesasExcel manual__2024-06-25T21:26:03.924987+00:00 [running]> on host 298ae34a2ca3
[2024-06-25T21:26:09.977+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='engenharia-dados' AIRFLOW_CTX_DAG_ID='sp_budget_pipeline' AIRFLOW_CTX_TASK_ID='ingestion-gdvDespesasExcel' AIRFLOW_CTX_EXECUTION_DATE='2024-06-25T21:26:03.924987+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-25T21:26:03.924987+00:00'
[2024-06-25T21:26:09.979+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T21:26:10.878+0000] {docker.py:366} INFO - Starting docker container from image etl-engine:latest
[2024-06-25T21:26:10.962+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-25T21:26:25.747+0000] {docker.py:436} INFO - 2024-06-25 21:26:25,747 [INFO] gdvDespesasExcel ingestion: Starting....
[2024-06-25T21:26:25.748+0000] {docker.py:436} INFO - 2024-06-25 21:26:25,747 [INFO] PROJECT_ID: homologation-310718 - CLIENT_EMAIL: datalake-hml@homologation-310718.iam.gserviceaccount.com
[2024-06-25T21:26:25.750+0000] {docker.py:436} INFO - 2024-06-25 21:26:25,749 [INFO] _read_schema
[2024-06-25T21:26:25.781+0000] {docker.py:436} INFO - 2024-06-25 21:26:25,781 [INFO] _enforce_schema
2024-06-25 21:26:25,781 [INFO] _read_schema
[2024-06-25T21:26:25.813+0000] {docker.py:436} INFO - 2024-06-25 21:26:25,812 [INFO] Running insert on table upload_manual.gdvdespesasexcel.
[2024-06-25T21:26:36.197+0000] {docker.py:436} INFO - 2024-06-25 21:26:36,196 [INFO] Insert on table upload_manual.gdvdespesasexcel finished!
[2024-06-25T21:26:36.199+0000] {docker.py:436} INFO - 2024-06-25 21:26:36,198 [INFO] gdvDespesasExcel ingestion: Done.
[2024-06-25T21:26:38.937+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T21:26:39.018+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sp_budget_pipeline, task_id=ingestion-gdvDespesasExcel, run_id=manual__2024-06-25T21:26:03.924987+00:00, execution_date=20240625T212603, start_date=20240625T212608, end_date=20240625T212639
[2024-06-25T21:26:39.080+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T21:26:39.110+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-25T21:26:39.155+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
